sentry-kotlin-compiler-plugin/build.gradle.kts

```diff
@@ -0,0 +1,41 @@
+plugins {
+    kotlin("jvm") version "1.8.20"
+    kotlin("kapt")
+    id("com.vanniktech.maven.publish") version "0.24.0"
```

Comment:
`m`: the rest of the repo is using version 0.17.0 because there were some [problems](https://github.com/getsentry/sentry-android-gradle-plugin/blob/7950250710c908293e9090b317f7bef473efa400/buildSrc/src/main/java/Dependencies.kt#L11-L13) with the way craft is consuming artifacts, which changed in 0.18.0. I guess we either will have to revert to 0.17.0 or adjust how we package artifacts.

Reply:
Unfortunately I can't because 0.17.0 does not provide the `mavenPublishing { coordinates() ... }` API